controller/worker/deployment/all_at_once.go

```diff
@@ -72,7 +72,7 @@ func (d *DeployJob) deployAllAtOnce() error {
 		log.Info("waiting for job events", "expected", expected)
 		if err := d.waitForJobEvents(d.OldReleaseID, expected, log); err != nil {
 			log.Error("error waiting for job events", "err", err)
-			return err
+			return ErrSkipRollback{err.Error()}
```

Comment:
Probably worth a comment (something along the lines of [this comment](https://github.com/flynn/flynn/issues/2162#issuecomment-157908326)).